.circleci/config.yml

```diff
@@ -204,10 +204,6 @@ jobs:
           keys:
             - v1-pkg-cache
       - run: go install github.com/mgechev/revive@latest
-      - run:
-          name: 🧹 Run Golang revive lint
-          command: |
-            sh ./check-revive.sh || exit 1
```

Comment:
Why is this block removed ?

Reply:
Ok I checked this was by mistake I selected this code also , I should not have , want me to add it back ?

Reply:
added